options.cpp

```diff
@@ -123,4 +135,16 @@ std::optional<std::filesystem::path> RtSmithOptions::p4InfoFilePath() const {
 
 std::string_view RtSmithOptions::controlPlaneApi() const { return _controlPlaneApi; }
 
+std::optional<std::filesystem::path> RtSmithOptions::fuzzerConfigPath() const {
+    return _fuzzerConfigPath;
+}
+
+std::optional<std::string> RtSmithOptions::fuzzerConfigString() const {
+    return _fuzzerConfigString;
+}
+
+void RtSmithOptions::setFuzzerConfigPath(const char *arg) { _fuzzerConfigPath = arg; }
```

Comment:
Do not use `char *arg`. You should only rarely have to use it in C++. Use `std::string_view`, `std::string`, or `std::filesystem::path` instead.